File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/FrontEndMigrationStatus.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

import org.taktik.icure.entities.base.EnumVersion;

import java.io.Serializable;

@EnumVersion(1l)
public enum FrontEndMigrationStatus implements Serializable {
    STARTED, ERROR, SUCCESS
}

File: src/main/kotlin/org/taktik/icure/entities/embed/FrontEndMigrationStatus.kt
package org.taktik.icure.entities.embed

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import org.taktik.icure.entities.base.EnumVersion
import java.io.Serializable

@EnumVersion(1L)
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
enum class FrontEndMigrationStatus : Serializable {
    STARTED, ERROR, SUCCESS
}
